x-pack/packages/ml/aiops_log_pattern_analysis/create_category_request.ts

```diff
@@ -38,7 +38,7 @@ export function createCategoryRequest(
   useStandardTokenizer: boolean = true,
   includeSparkline: boolean = true
 ) {
-  const query = createCategorizeQuery(queryIn, timeField, timeRange);
+  const query = createDefaultQuery(queryIn, timeField, timeRange);
```

Comment:
If I have a filter as well as the `*` query, it looks like the filter is getting ignored (happens in the AIOps Labs pages for Pattern Analysis and Change Point, and in Discover Patterns tab):

<img width="1000" alt="Screenshot 2024-10-04 at 16 53 55" src="https://github.com/user-attachments/assets/f5687862-7363-45cd-8834-d8abf7eed0f7">

Remove the `*` query and the correct filtering is applied:

<img width="1202" alt="Screenshot 2024-10-04 at 16 56 04" src="https://github.com/user-attachments/assets/f6be0e71-c687-45a5-8ad7-fef60b4631b4">

Reply:
This is an unrelated issue to the fix in this PR and I suspect it'll require a bit of a rewrite of how we're retrieving query items.
I'll keep looking at it and hopefully I can include it in this PR. But if it ends up being too large I think we could merge this PR as is, so the `*` fix makes it into the serverless release and then get the filters fix in asap afterwards.

Reply:
Fix in a separate PR if this makes sense.

Reply:
I've created an issue for this https://github.com/elastic/kibana/issues/195212